integrations/docker/images/stage-2/chip-build-cirque/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=latest
+ARG VERSION=15
```

Comment:
Could you explain the cirque logic? I would have expected that if this is in stage-build we would instead do whatever others are doing (and then not lock in the version).

Why aren't we changing https://github.com/project-chip/connectedhomeip/blob/master/.github/workflows/cirque.yaml#L87 to run the cirque image that is fixed and not rebuild it every time?

If we are looking for reproducible builds, ideally images should be fixed. I believe at some point I was told that this is due to OTBR .... is that true? Could you explain and maybe we should have that explanation as a comment in cirque.yaml ... or better define if this is not needed have cirque.yaml fix the version.